### PR TITLE
[Tool]  build with clang13

### PR DESCRIPTION
**Pull request by murphyatwork**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [x] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

File: thirdparty/build-thirdparty.sh [hunk 1]
```diff
@@ -467,8 +467,8 @@ build_brpc() {
     check_if_source_exist $BRPC_SOURCE
 
     cd $TP_SOURCE_DIR/$BRPC_SOURCE
-    ./config_brpc.sh --headers="$TP_INSTALL_DIR/include /usr/include" --libs="$TP_INSTALL_DIR/bin $TP_INSTALL_DIR/lib /usr/lib" --with-glog
```
Comment (imay) [inline thirdparty/build-thirdparty.sh:470]: Why do you remove the glog?
We need glog to print brpc's log.

File: thirdparty/build-thirdparty.sh [hunk 2]
```diff
@@ -978,7 +978,6 @@ build_boost # must before thrift
 build_protobuf
 build_gflags
 build_gtest
-build_glog
```
Comment (imay) [inline thirdparty/build-thirdparty.sh:981]: why do you remove glog? we needs it

File: thirdparty/build-thirdparty.sh (hunk 1 above)
Comment (mofeiatwork) [reply thirdparty/build-thirdparty.sh:470]: fixed

**Comment by mofeiatwork** [comment]
run starrocks_admit_test

File: thirdparty/build-thirdparty.sh [hunk 3]
```diff
@@ -468,7 +468,7 @@ build_brpc() {
 
     cd $TP_SOURCE_DIR/$BRPC_SOURCE
     ./config_brpc.sh --headers="$TP_INSTALL_DIR/include /usr/include" --libs="$TP_INSTALL_DIR/bin $TP_INSTALL_DIR/lib /usr/lib" --with-glog
-    ${BUILD_SYSTEM} -j$PARALLEL
+    make -j$PARALLEL
```
Comment (satanson) [inline thirdparty/build-thirdparty.sh:471]: not support ninja ？

**Review by imay — APPROVED** [review]

**Pull request merged**